plugins/generator-1.15.2/forge-1.15.2/procedures/entity_is_elytra_flying.ftl

```diff
@@ -0,0 +1 @@
+(((PlayerEntity) entity).isElytraFlying())
```

Comment:
Add support to change the entity variable (check other entity blocks)

Reply:
oh i have forgot it :D

Reply:
Cast check is missing too, applies to all other blocks

Reply:
i have forgot to delete the old procedures

Reply:
mean you (int) 1.0F... , with cast check?

Reply:
it's the `if (${input$entity} instanceof PlayerEntity)` part, to make sure that the passed entity is actually a player

Reply:
thx its right now fixed